libs/coin-modules/coin-stacks/package.json

```diff
@@ -0,0 +1,136 @@
+{
+  "name": "@ledgerhq/coin-stacks",
+  "version": "0.1.0",
+  "description": "Ledger Stacks Coin integration",
+  "keywords": [
+    "Ledger",
+    "LedgerWallet",
+    "stacks",
+    "stx",
+    "Bitcoin layer2",
+    "Hardware Wallet"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LedgerHQ/ledger-live.git"
+  },
+  "bugs": {
+    "url": "https://github.com/LedgerHQ/ledger-live/issues"
+  },
+  "homepage": "https://github.com/LedgerHQ/ledger-live/tree/develop/libs/coin-modules/coin-stacks",
+  "publishConfig": {
+    "access": "public"
+  },
+  "typesVersions": {
+    "*": {
+      "lib/*": [
+        "lib/*"
+      ],
+      "lib-es/*": [
+        "lib-es/*"
+      ],
+      "deviceTransactionConfig": [
+        "lib/bridge/deviceTransactionConfig"
+      ],
+      "specs": [
+        "lib/test/bot-specs"
+      ],
+      "transaction": [
+        "lib/bridge/transaction"
+      ],
+      "*": [
+        "lib/*"
+      ]
+    }
+  },
+  "exports": {
+    "./lib/*": "./lib/*.js",
+    "./lib-es/*": "./lib-es/*.js",
+    "./api": {
+      "require": "./lib/api/index.js",
+      "default": "./lib-es/api/index.js"
+    },
+    "./deviceTransactionConfig": {
+      "require": "./lib/bridge/deviceTransactionConfig.js",
+      "default": "./lib-es/bridge/deviceTransactionConfig.js"
+    },
+    "./signer": {
+      "require": "./lib/signer/index.js",
+      "default": "./lib-es/signer/index.js"
+    },
+    "./hw-signMessage": {
+      "require": "./lib/signer/signMessage.js",
+      "default": "./lib-es/signer/signMessage.js"
+    },
+    "./specs": {
+      "require": "./lib/test/bot-specs.js",
+      "default": "./lib-es/test/bot-specs.js"
+    },
+    "./transaction": {
+      "require": "./lib/bridge/transaction.js",
+      "default": "./lib-es/bridge/transaction.js"
+    },
+    "./types": {
+      "require": "./lib/types/index.js",
+      "default": "./lib-es/types/index.js"
+    },
+    "./*": {
+      "require": "./lib/*.js",
+      "default": "./lib-es/*.js"
+    },
+    ".": {
+      "require": "./lib/index.js",
+      "default": "./lib-es/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@ledgerhq/coin-framework": "workspace:^",
+    "@ledgerhq/cryptoassets": "workspace:^",
+    "@ledgerhq/devices": "workspace:*",
+    "@ledgerhq/errors": "workspace:^",
+    "@ledgerhq/live-env": "workspace:^",
+    "@ledgerhq/live-network": "workspace:^",
+    "@ledgerhq/logs": "workspace:^",
+    "@ledgerhq/types-cryptoassets": "workspace:^",
+    "@ledgerhq/types-live": "workspace:^",
+    "axios": "1.7.7",
+    "bignumber.js": "^9.1.2",
+    "c32check": "1.1.3",
+    "invariant": "^2.2.4",
+    "lodash": "^4.17.21",
+    "ripple-address-codec": "^5.0.0",
+    "ripple-binary-codec": "^1.3.0",
+    "rxjs": "^7.8.1",
+    "@stacks/network": "6.10.0",
+    "@stacks/transactions": "6.11.0",
+    "@zondax/ledger-stacks": "^1.0.2"
```

Comment:
This is only related to nano app right ? shouldn't be in coin-module

Reply:
It is used in `signer.ts` for the interface `StacksSigner` I am moving it in `devDependencies`